lib/remotable/nosync.rb

```diff
@@ -19,13 +19,19 @@ def nosync=(val)
     end
 
     def nosync_value?
-      !@nosync.nil?
+      !_nosync.nil?
     end
 
     def nosync?
       @nosync == true
     end
 
+  private
+
+    def _nosync
+      @nosync if instance_variable_defined?(:@nosync)
+    end
```

Comment:
What does this help with?

Reply:
We were getting the warning `instance variable @nosync was not defined` when we were checking if `@nosync` was nil. We're OK with `@nosync` being `nil`, but to avoid the warning, I encapsulated accessing the instance variable in a method that'll return nil if it isn't defined.

Reply:
👍